cmd/kubelet/app/server.go

```diff
@@ -297,8 +297,7 @@ func initConfigz(kc *componentconfig.KubeletConfiguration) (*configz.Config, err
 }
 
 func run(s *options.KubeletServer, kubeDeps *kubelet.KubeletDeps) (err error) {
-	// TODO: this should be replaced by a --standalone flag
-	standaloneMode := (len(s.APIServerList) == 0 && !s.RequireKubeConfig)
+	standaloneMode := s.Standalone
```

Comment:
I assume it does. Probably we can guard this by putting an assert that verifies that APIServerList is empty when running in standalone mode. Do you think this would be a acceptable solution?